bedrock/mozorg/templates/mozorg/antiharassment-tool.html

```diff
@@ -0,0 +1,210 @@
+    <h2>More than half of all people have experienced online harassment, cyberstalking, doxxing, or online violence.</h2>
+    <p>Women, people of color, and the LBGTQ+ community are disproportionately affected. People are also constantly targeted
+    due to their line of work, their professional opinions, and their identities.</p>
+    <p><strong>This is not okay and we are here to help put an end to it.</strong></p>
+    <div class="c-button-group">
+      <a href="#newsletter" class="mzp-c-button c-button-aht-secondary-inverse">Get product updates</a>
+      <a href="https://mozilla.formstack.com/forms/online_harassment" rel="external noopener" target="_blank" class="mzp-c-button c-button-aht-primary-inverse">Test our product</a>
+    </div>
+  </div>
+</section>
+
+<section class="mzp-l-content mzp-t-content-lg">
+  <h2 class="section-heading">Our vision</h2>
+  <h3 class="section-subheading">We’re currently building tools to help keep people safe online.</h3>
+  <p>Our goal is to fight for wellbeing, promote digital safety, and empower people to do their jobs without being harassed.
+  We want to help people who’ve experienced online harassment move from a place of fear, to a place of resilience.</p>
+</section>
+
+<section class="c-help-section">
+  <div class="mzp-l-content mzp-t-content-lg">
+    <h2 class="section-heading">We need your help to build the best solution possible</h2>
+    <h3 class="section-subheading">Throughout this exploration, we’ve been talking to real people who are facing this problem to ensure we build products
+    responsibly, and with your needs at the center.</h3>
+    <p>If you've experienced online harassment due to your line of work, your professional opinions, or your identity, we want
+    to hear from you. (And we’re sorry you’ve had to go through this). Your unique perspective can help us ensure that our
+    products will make a meaningful difference.</p>
+    <p>This may involve a survey, a 1-on-1 interview, or viewing a live prototype.
+    Your responses are anonymous and we’ll compensate you for your time.</p>
+    <a href="https://mozilla.formstack.com/forms/online_harassment" rel="external noopener" target="_blank" class="mzp-c-button c-button-aht-secondary">Help test our product</a>
+  </div>
+</section>
+
+<section class="c-resources-section mzp-l-content mzp-t-content-lg">
+  <h2 class="section-heading">Resources for online harassment</h2>
+  <p>Mozilla is a member of the <a href="https://onlineviolenceresponsehub.org/" rel="external noopener" target="_blank">Coalition Against Online Violence</a>, which helps women journalists facing online abuse. If
+  you’re currently facing online harassment of other forms of online violence, here are more organizations we recommend.</p>
+  <div class="c-resource-grid">
+    <div class="c-resource-grid-item">
+      <a href="https://cybercivilrights.org/ccri-crisis-helpline/" rel="external noopener" target="_blank">CCRI Crisis Helpline</a>
```

Comment:
**Suggestion (design/non-blocking):** These headers for this section don't really give the impression that they also may be hyperlinks? Maybe we can add an underline to suggest that? I think we should get Jessica's opinion here for how to design it to make it look like an actionable item while also acting like a header? 
Maybe we can add a hyperlink icon next to them...?